Static/Row.swift

```diff
@@ -145,9 +148,10 @@ public struct Row: Hashable, Equatable {
 
     // MARK: - Initializers
 
-    public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
+    public init(accessibilityIdentifier: String? = nil, text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
```

Comment:
What do you think about moving this parameter to the end? 

I think for "future compatibility" sake, adding optional parameters to the end of a series of initializers clarifies usage for those upgrading from one version to the next.

Reply:
Great point, thanks!